inference-engine/tests/unit/inference_engine/ie_blob_proxy.cpp

```diff
@@ -28,27 +27,22 @@ class BlobProxyTests: public ::testing::Test {
     shared_ptr<MockAllocator> createMockAllocator() {
         return shared_ptr<MockAllocator>(new MockAllocator());
     }
-
-
-public:
-
 };
 
 TEST_F(BlobProxyTests, convertByteBlobToFloat) {
     const int size = 4;
     float test_array[size] = {2.2f, 3.5f, 1.1f, 0.0f};
     TBlob<uint8_t>::Ptr b(new TBlob<uint8_t>({ Precision::U8, {size * sizeof(float)}, C }));
     b->allocate();
-    uint8_t *sPtr = (uint8_t *) test_array;
+    uint8_t *sPtr = reinterpret_cast<uint8_t *>(test_array);
     uint8_t *dPtr = b->data();
     ASSERT_EQ(b->size(), size * sizeof(float));
     for (size_t i = 0; i < b->size(); i++) {
         dPtr[i] = sPtr[i];
     }
 }
 
-TEST_F(BlobProxyTests, shouldNotDeAllocate)
-{
+TEST_F(BlobProxyTests, shouldNotDeAllocate) {
```

Comment:
From gtest ideology perspective, usage of `TEST_F` here is redundant - the class for which you are instantiating the tests doesn't create any fixtures.  I propose to get rid of `BlobProxyTests` class and use simple `TEST` macro to define tests.

As far as I understand, the only reasons to have a test class is to implement `createMockAllocator()`  function, which is one line anyway and mock object can be created in each test without having a class

Reply:
Updated